python/paddle/nn/functional/common.py

```diff
@@ -2081,67 +2082,64 @@ class centers and the shape of sampled_class_center will be [num_positive_class_
+        >>> num_samples = 6
+        >>> label = paddle.randint(low=0, high=num_classes, shape=[batch_size], dtype='int64')
+        >>> remapped_label, sampled_class_index = paddle.nn.functional.class_center_sample(label, num_classes, num_samples)
+
+        >>> print(label)
+        >>> print(remapped_label)
+        >>> print(sampled_class_index)
+        >>> # the output is
+        Tensor(shape=[10], dtype=int64, place=Place(cpu), stop_gradient=True,
+        [10, 3 , 11, 14, 15, 15, 7 , 12, 5 , 13])
+        Tensor(shape=[10], dtype=int64, place=Place(cpu), stop_gradient=True,
+        [3, 0, 4, 7, 8, 8, 2, 5, 1, 6])
+        Tensor(shape=[9], dtype=int64, place=Place(cpu), stop_gradient=True,
+        [3 , 5 , 7 , 10, 11, 12, 13, 14, 15])
 
     .. code-block:: python
         :name: code-example2
 
-        # required: distributed
-        # Multi GPU, test_class_center_sample.py
-        import paddle
-        import paddle.distributed as dist
-        strategy = dist.fleet.DistributedStrategy()
-        dist.fleet.init(is_collective=True, strategy=strategy)
-        batch_size = 10
-        num_samples = 6
-        rank_id = dist.get_rank()
-        # num_classes of each GPU can be different, e.g num_classes_list = [10, 8]
-        num_classes_list = [10, 10]
-        num_classes = paddle.sum(paddle.to_tensor(num_classes_list))
-        label = paddle.randint(low=0, high=num_classes.item(), shape=[batch_size], dtype='int64')
-        label_list = []
-        dist.all_gather(label_list, label)
-        label = paddle.concat(label_list, axis=0)
-        remapped_label, sampled_class_index = paddle.nn.functional.class_center_sample(label, num_classes_list[rank_id], num_samples)
-
-        print(label)
-        print(remapped_label)
-        print(sampled_class_index)
-
-        #python -m paddle.distributed.launch --gpus=0,1 test_class_center_sample.py
-        # rank 0 output:
-        #Tensor(shape=[20], dtype=int64, place=CUDAPlace(0), stop_gradient=True,
-        #       [10, 17, 15, 11, 9 , 12, 18, 18, 17, 18, 19, 2 , 8 , 13, 11, 13, 9 , 10, 0 , 4 ])
-        #Tensor(shape=[20], dtype=int64, place=CUDAPlace(0), stop_gradient=True,
-        #       [6 , 11, 10, 7 , 4 , 8 , 12, 12, 11, 12, 13, 1 , 3 , 9 , 7 , 9 , 4 , 6 , 0 , 2 ])
-        #Tensor(shape=[6], dtype=int64, place=CUDAPlace(0), stop_gradient=True,
-        #       [0, 2, 4, 8, 9, 3])
-
-        # rank 1 output:
-        #Tensor(shape=[20], dtype=int64, place=CUDAPlace(1), stop_gradient=True,
-        #       [10, 17, 15, 11, 9 , 12, 18, 18, 17, 18, 19, 2 , 8 , 13, 11, 13, 9 , 10, 0 , 4 ])
-        #Tensor(shape=[20], dtype=int64, place=CUDAPlace(1), stop_gradient=True,
-        #       [6 , 11, 10, 7 , 4 , 8 , 12, 12, 11, 12, 13, 1 , 3 , 9 , 7 , 9 , 4 , 6 , 0 , 2 ])
-        #Tensor(shape=[7], dtype=int64, place=CUDAPlace(1), stop_gradient=True,
-        #       [0, 1, 2, 3, 5, 7, 8])
+        >>> # required: distributed
+        >>> # Multi GPU, test_class_center_sample.py
```

Comment:
好像依赖外部文件 `test_class_center_sample.py`，这段代码skip吧

```suggestion
        >>> # doctest: +SKIP('Depends on external files.')
        >>> # required: distributed
        >>> # Multi GPU, test_class_center_sample.py
```

Reply:
这里改用 `# doctest: +REQUIRES(env:DISTRIBUTED)` 即可，因为我们 CI 没有分布式环境，使用该 requires 就是自动跳过